builder/main.py

```diff
@@ -162,41 +165,40 @@ def parse_args():
             print('No command provided, should be [build|inspect|<action-name>]')
         sys.exit(1)
 
+    # parse the args we know, the rest end up in args.args for others to parse
+    args = parser.parse_args(argv)
```

Comment:
This should be parse_known_args()

Reply:
There are no unknown arguments because of this line (a few lines up):
`parser.add_argument('args', nargs=argparse.REMAINDER)`

that causes all unknown arguments to end up in `args.args`.

The old way did `args, extra = parser.parse_unknown_args()` but it was pointless, `extra` was always empty.

Reply:
I guess I'm saying, we should use that argparse.REMAINDER thing, or use parse_unknown_args(), but doing both is pointless and confusing

Reply:
oh I see, let's get rid of the remainder then, because actions depend on being able to find their args in there without modifying main